.github/CODEOWNERS

```diff
@@ -94,6 +99,10 @@
 /kernel/byterun/  @maximedenes
 # Secondary maintainer @silene
 
+/kernel/uGraph.ml{,i} @SkySkimmer
+/kernel/univ.ml{,i} @SkySkimmer
+# Secondary maintainer @mattam82
```

Comment:
sorts.ml is also universe stuff btw

Reply:
Oh yes, of course... let me update this.